src/features/embalm/stepContent/mocks/mockArchaeologists.ts

```diff
@@ -17,12 +17,15 @@ export function mochArchaeologists(numberOfArcheologists: number): Archaeologist
     let archProfile: ArchaeologistProfile = {
       archAddress: '0x00000000000000000000000000000000000' + randomNumber.toString(),
       exists: true,
-      minimumDiggingFee: BigNumber.from(
+      minimumDiggingFeePerSecond: BigNumber.from(
```

Comment:
I wonder if we need `mockArchaeologists.ts` at all any longer

Reply:
We certainly don't, I just haven't taken the time to remove it yet.